format/spec.md

```diff
@@ -1003,6 +1032,7 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
 | **`uuid`**         | `fixed_len_byte_array[16]`                                         | `UUID`                                      |                                                                |
 | **`fixed(L)`**     | `fixed_len_byte_array[L]`                                          |                                             |                                                                |
 | **`binary`**       | `binary`                                                           |                                             |                                                                |
+| **`variant`**      | `group` containing fields according to the Variant spec            |                                             |                                                                |
```

Comment:
I think for compatibility with other engines this may need a Logical Type as well